src/js/components/chat-input.jsx

```diff
@@ -7,6 +7,8 @@ import debounce from 'lodash.debounce';
 import { sendMessage, resetUnreadCount } from 'services/conversation-service';
 import { store } from 'stores/app-store';
 
+import { ImageUpload } from 'components/image-upload';
```

Comment:
Because I built it that way before we even started on the web app. We could go through everything and fix it though, but I don't think it's necessary in this PR.